### PR TITLE
[MIG] vendor_transport_lead_time: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**

### CODE REVIEW

File: vendor_transport_lead_time/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Vendor transport lead time",
     "summary": "Purchase delay based on transport and supplier delays",
-    "version": "13.0.1.2.0",
+    "version": "14.0.1.2.0",
```
Comment (simahawk) [inline vendor_transport_lead_time/__manifest__.py]: 1.0.0

**Review by simahawk — COMMENTED** [review]
@phuctranerp can you squash fixup commits?  I know they were kept by error but is not nice to carry them on from version to version. Thanks! :pray: 

FTR we are experiencing perf issue w/ this module on big POs because the Odoo core machinery to select seller is freaking slow. We'll have to invent something to work around it.

File: vendor_transport_lead_time/__manifest__.py (hunk 1 above)
Comment (phuctranfxvn) [reply vendor_transport_lead_time/__manifest__.py]: fixed

**Comment by mmequignon** [comment]
@phuctranerp You lost some informations in the process, I think, at least in the contributors.
@simahawk What you asked has been fixed

**Comment by simahawk** [comment]
pls squash all commits from @mmequignon :smile:

**Comment by mmequignon** [comment]
> pls squash all commits from @mmequignon smile

:fearful:

**Comment by hailangvn** [comment]
Good day @mmequignon, @simahawk, the missing information was removed by oca-bot because that information is not in `readme/*` files (https://github.com/OCA/purchase-workflow/commit/955226defe72343ccddb15b90c01cb9a7947781f). The information is now retained in squashed commits. Thanks for your support.

**Review by mmequignon — APPROVED** [review]

**Comment by hailangvn** [comment]
This has been rebased to latest 14.0.

**Review by sebalix — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**